pkg/apis/upgrade/v1alpha1/upgradeconfig_types.go

```diff
@@ -25,6 +25,7 @@ type UpgradeConfigSpec struct {
 	// Specify the upgrade start time
 	UpgradeAt string `json:"upgradeAt"`
 
+	// +kubebuilder:validation:Minimum:=0
 	// The maximum grace period granted to a node whose drain is blocked by a Pod Disruption Budget, before that drain is forced. Measured in minutes.
```

Comment:
I have one small request, could you also update the comment description here to state what a '0' means in the context of this field and how MUO behaves? So that we don't have to wonder in the future what MUO does with a '0' value? :)  (if my memory is correct from your previous testing that meant that MUO would grant no grace period at all)

Reply:
Thanks for the review @mrbarge  The minimum value here to just restrict and not allowing any negative integer value for this param. Also "0" value is currently not respecting `nodeDrain.expectedNodeDrainTime` set in the configmap and executing pdb delete strategy immediately. This issue is tracked here [OSD-7899](https://issues.redhat.com/browse/OSD-7899).
Still i am not clear that it should respect nodeDrain.expectedNodeDrainTime if PDBForceDrainTimeout<= nodeDrain.expectedNodeDrainTime Or just for "0" value.

Reply:
Oh yeah, understood - to clarify my request, I meant updating the actual field comment in the code of what "0" means in terms of behaviour (so that it also gets reflected in the CRD field description  :)  ). 
The change looks good to me otherwise.

Reply:
Thanks for the change :)
/lgtm